objectbox/lib/src/annotations.dart

```diff
@@ -74,11 +104,16 @@ enum PropertyType {
   // stringVector
 }
 
+/// Annotation Id can be used to specify an entity ID property if it's named
+/// anything else then "id" (case insensitive).
```

Comment:
```suggestion
/// anything else than "id" (case insensitive).
```
But "named anything else than" sounds wrong to me anyhow. Not sure how to say this properly though.